uberfire-experimental/uberfire-experimental-backend/src/test/java/org/uberfire/experimental/service/BackendExperimentalFeaturesRegistryServiceImplTest.java

```diff
@@ -241,31 +171,20 @@ private void testStoreFeature(String featureId, boolean newValue, ExperimentalFe
                 .hasFieldOrPropertyWithValue("enabled", newValue);
     }
 
-    private void init(Boolean enableExperimental, boolean loadData) {
-        if (loadData) {
-            try {
-                Path path = fileSystem.getPath(globalStorage.getStoragePath());
-                ioService.write(path, IOUtils.toString(getClass().getResourceAsStream("/test/global/regularFeatures.txt"), Charset.defaultCharset()));
+    private void init(Boolean enableExperimental) {
+
+        System.setProperty(BackendExperimentalFeaturesRegistryServiceImpl.EXPERIMENTAL_FEATURES_PROPERTY_NAME, enableExperimental.toString());
```

Comment:
I would add an `@After` method which cleanup the property.

Reply:
oh thanks!